### PR TITLE
feat: adds mobile onboarding modal

### LINKED ISSUE #452: Create onboarding for mobile users

### PR DESCRIPTION
**Pull request by kenodressel**
the link is subject to change when we have a correct one. Everything else can be reviewed :) fixes #452

### CODE REVIEW

File: src/components/OnboardingModal.vue [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        {{ $t('onboardingModal.headline') }}
+      </h1>
+      <p class="info-text">
+        {{ $t('onboardingModal.body') }}
+      </p>
+      <p class="info-text">
+        <a
+          href="https://dex.superhero.com"
+          target="_blank"
+        >https://dex.superhero.com</a>
```
Comment (CedrikNikita) [inline src/components/OnboardingModal.vue:18]: Shall we use aepp.dex.superhero.com instead?

File: src/components/OnboardingModal.vue [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  },
+  props: {
+    resolve: { type: Function, required: true },
+    reject: { type: Function, required: true },
+  },
+  methods: {
+    close() {
+      this.resolve();
+    },
+  },
```
Comment (CedrikNikita) [inline src/components/OnboardingModal.vue:48]:
```suggestion
```
Can we just resolve it on click and not wrap it with method?

Comment (kenodressel) [reply src/components/OnboardingModal.vue]: fixed :)

File: src/components/OnboardingModal.vue (hunk 1 above)
Comment (kenodressel) [reply src/components/OnboardingModal.vue:18]: the url is not final at the moment but will be basically leading to an FAQ page

Comment (kenodressel) [reply src/components/OnboardingModal.vue]: final url is in now

File: src/components/OnboardingModal.vue [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        {{ $t('onboardingModal.headline') }}
+      </h1>
+      <p class="info-text">
+        {{ $t('onboardingModal.body') }}
+      </p>
+      <p class="info-text">
+        <a
+          href="https://superherodex.mimosa.digital/superhero-dex-and-ios"
+          target="_blank"
+        >https://superherodex.mimosa.digital/superhero-dex-and-ios</a>
```
Comment (CedrikNikita) [inline src/components/OnboardingModal.vue:18]:
```suggestion
        >
          https://superherodex.mimosa.digital/superhero-dex-and-ios
        </a>
```

**Review by CedrikNikita — APPROVED** [review]
Looks OK from code perspective.

File: src/components/OnboardingModal.vue (hunk 3 above)
Comment (kenodressel) [reply src/components/OnboardingModal.vue]: thx!

**Pull request merged**